packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -662,7 +674,7 @@ class InkResponse extends StatelessWidget {
       enableFeedback: enableFeedback,
       excludeFromSemantics: excludeFromSemantics,
       focusNode: focusNode,
-      canRequestFocus: canRequestFocus,
+      focusable: focusable && canRequestFocus,
```

Comment:
What if one is false and the other is true?

Reply:
What I figured is that if either case is `false`, the result will be `false`. This simplifies the four possible scenarios and prevents anyone using it from being surprised. Also, considering that the intention is to completely replace and not use both options.

Reply:
The feedback about why we need to make this change has not been addressed yet. Without a reason, we will likely close this PR.